test/powershell/Modules/Microsoft.WinGet.CommandNotFound/Microsoft.WinGet.CommandNotFound.Tests.ps1

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+Describe "Microsoft.WinGet.CommandNotFound" -tags "CI" {
+    BeforeAll {
+        if (Get-Module Microsoft.WinGet.CommandNotFound) {
+            Remove-Module Microsoft.WinGet.CommandNotFound
+        }
+    }
+
+    Context "Windows only" {
+        It "Should import the module correctly" -Skip:(-not $IsWindows) {
+            Import-Module -Name Microsoft.WinGet.CommandNotFound
+            $module = Get-Module Microsoft.WinGet.CommandNotFound
+            $module.Name | Should -BeExactly 'Microsoft.WinGet.CommandNotFound'
+            $module.Version | Should -Match '^1.0.4$'
+        }
+
+        It "Should be installed to `$PSHOME" -Skip:(-not $IsWindows) {
+            $module = Get-Module (Join-Path -Path $PSHOME -ChildPath "Modules" -AdditionalChildPath "Microsoft.WinGet.CommandNotFound") -ListAvailable
+            $module.Name | Should -BeExactly 'Microsoft.WinGet.CommandNotFound'
+            $module.Version | Should -Match '^1.0.4$'
+            $module.Path | Should -Be (Join-Path -Path $PSHOME -ChildPath "Modules/Microsoft.WinGet.CommandNotFound/Microsoft.WinGet.CommandNotFound.psd1")
+        }
+    }
+
+    Context "Linux and macOS only" {
+        It "Should not be installed to `$PSHOME" -Skip:($IsWindows) {
+            Test-Path (Join-Path -Path $PSHOME -ChildPath "Modules" -AdditionalChildPath "Microsoft.WinGet.CommandNotFound") | Should -Be $False
+        }
+    }
+
```

Comment:
Yeah! So that's something I wanted some guidance on, actually. Idk how to test the FeedbackProvider, so no functional tests exist in this repo or over on [microsoft/winget-command-not-found](https://github.com/microsoft/winget-command-not-found).

I'd be happy to add some tests, if that's desired. But the codeflow is just grabbing the missing command from the FeedbackProvider subsystem, [throwing it over to winget to search for packages](https://github.com/microsoft/winget-command-not-found/blob/9bc83617b94f6dc88e1fc9599e1c859bc3adf96f/src/WinGetCommandNotFoundFeedbackPredictor.cs#L147-L209), then displaying the suggested packages using the FeedbackProvider subsystem.